README.md

```diff
@@ -1 +1,20 @@
-# wavefront-go-middleware
\ No newline at end of file
+# wavefront-go-middleware [![travis build status](https://travis-ci.com/wavefrontHQ/wavefront-go-middleware.svg?branch=master)](https://travis-ci.com/wavefrontHQ/wavefront-go-middleware) [![Go Report Card](https://goreportcard.com/badge/github.com/wavefrontHQ/wavefront-go-middleware)](https://goreportcard.com/report/github.com/wavefrontHQ/wavefront-go-middleware) [![OpenTracing Badge](https://img.shields.io/badge/OpenTracing-enabled-blue.svg)](http://opentracing.io)
+
+This library provides wavefront-opentracing-middleware support for various Go's web framework. The middleware takes care of the entire lifecycle of span reporting with minimal code injection in Api's. It provides a custom logger abstracted on top of Go standard Logger which injects request-scoped unique trace identifiers into logs generated while servicing an Api request. 
```

Comment:
Some minor nits: 
* for various Go web frameworks.
* The middleware handles the entire lifecycle of reporting spans with minimal code injection within APIs.